secretstores/hashicorp/vault/vault.go

```diff
@@ -40,6 +43,12 @@ const (
 	defaultVaultKVPrefix         string = "dapr"
 	vaultHTTPHeader              string = "X-Vault-Token"
 	vaultHTTPRequestHeader       string = "X-Vault-Request"
+	vaultEnginePath              string = "enginePath"
+	parseAsMap                   string = "parseAsMap"
```

Comment:
Can these constants have a better name? Just naming it as the content will satisfy linter but does not really improve readability.